naptime/src/main/scala/org/coursera/naptime/ari/engine/Utilities.scala

```diff
@@ -36,51 +36,94 @@ import scala.collection.JavaConverters._
 
 
 object Utilities extends StrictLogging {
-
-  // TODO(bryan): Dedupe this from NaptimeUnionField
```

Comment:
is this todo still relevant? should we file an issue/jira for it if its still an issue?

Reply:
It is no longer relevant, the logic is different from `NaptimeUnionField`